less-discourse/README.md

```diff
@@ -0,0 +1,31 @@
+
+### Menu on the Discourse Forum
+
+The goal was to rebuild the exact same menu in both UI and on the Discourse Forum.
+Only a few icons change (to differenciate between the already-too-similar-ones on Discourse) and the HTML for the menu on Discourse had to be adapted.
+
+If you make any changes to the `sidemenu.html` or `sidemenu.less`, make sure you update the Discourse's menu too.
```

Comment:
indeed i meant the original sidemenu template, nothing to do with menu.html. I will use full paths.